docs/index.html

```diff
@@ -139,7 +141,9 @@ <h1 id="iban-info-title" class="text-2xl font-bold text-primary mb-4">IBAN Infor
                 skipToMain: 'Skip to main content',
                 langChanged: 'Language changed to English.',
                 formReset: 'Form has been reset.',
-                disclaimer: 'Disclaimer: This website does not process payments or donations. It only helps you copy IBAN information to use in your banking app or other payment methods for actual transfers.'
+                disclaimer: 'Disclaimer: This website does not process payments or donations. It only helps you copy IBAN information to use in your banking app or other payment methods for actual transfers.',
+                nameSuggestion: 'Tip: Avoid special characters (ă, ș, ț, etc.) for maximum compatibility with all banks.',
+                detailsSuggestion: 'Tip: Avoid special characters (ă, ș, ț, etc.) for maximum compatibility with all banks.'
```

Comment:
I want to make it clear that the exact same text is used for more fields, so remove these duplications by using a broader key name like `specialCharactersSuggestion`